### ISSUE TITLE
"Crossfire-response-badrequest-title": "The URL is invalid."

### ISSUE THREAD

**Issue by jpawlyn** [fastlane/itc-api-docs#8]
I started receiving the error
```
{
  "itcBaseUrl": "https://itunesconnect.apple.com",
  "message": "Make sure the URL is correct and try again.",
  "l10n": {
    "Crossfire-response-badrequest-title": "The URL is invalid.",
    "Crossfire-response-badrequest-message": "Make sure the URL is correct and try again."
  },
  "errors": []
}
```
recently. Turns out that when making requests to `analytics.itunes.apple.com` there now needs to be the HTTP header:
```
'X-Requested-By' => 'analytics.itunes.apple.com'
```

**Comment by KrauseFx** [comment]
Thanks for letting me know. But the docs don't cover analytics yet.

**Comment by jpawlyn** [comment]
Good point and apologies, this certainly doesn't count as an issue. Was just wanting to highlight the problem that came up a week ago for those who use your login method and then access any of the analytics calls.

**Comment by simongcx** [comment]
Hi Jolyon,
I'm getting the same error as you, but the 'X-Requested-By' tip hasn't fixed it. Would you mind sharing your code?
Thanks!

**Comment by jpawlyn** [comment]
Afraid I'm unable to share but happy to take a look at your's if you would like to email it to me at gmail (jpawlyn).

**Comment by iHTCboy** [comment]
2021: 

host: https://appstoreconnect.apple.com

'X-Requested-By' => 'analytics.itunes.apple.com'

change to：

'X-Requested-By' => 'dev.apple.com'
